src/main/java/com/byznass/tiolktrack/postgres/dao/PostgresGpsProvider.java

```diff
@@ -0,0 +1,67 @@
+package com.byznass.tiolktrack.postgres.dao;
+
+import com.byznass.tiolktrack.kernel.TiolkTrackException;
+import com.byznass.tiolktrack.kernel.dao.GpsProvider;
+import com.byznass.tiolktrack.kernel.dao.LocationProvider;
+import com.byznass.tiolktrack.kernel.dao.NoGpsWithIdException;
+import com.byznass.tiolktrack.kernel.model.Gps;
+import com.byznass.tiolktrack.kernel.model.Location;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+public class PostgresGpsProvider implements GpsProvider {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(PostgresGpsProvider.class);
+
+	private final Connection connection;
+	private final LocationProvider locationProvider;
+
+	@Inject
+	public PostgresGpsProvider(Connection connection, LocationProvider locationProvider) {
+
+		this.connection = connection;
+		this.locationProvider = locationProvider;
+	}
+
+	@Override
+	public Gps getGpsById(String gpsId) throws NoGpsWithIdException, TiolkTrackException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.dao.NoGpsWithIdException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.TiolkTrackException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)